amethyst_core/src/transform/components/parent.rs

```diff
@@ -0,0 +1,13 @@
+
+use specs::{Component, DenseVecStorage, Entity, FlaggedStorage};
+
+/// Component for defining a parent entity.
```

Comment:
This documentation should exactly document on which entity one has to place the component. (It's confusing whether this is a "has" relation or an "is parent of".

Reply:
Rename it to `HasParent` maybe ?

Reply:
Or `ChildOf`

Reply:
This was addressed.